org.koromelodev.mindmate.yml

```
@@ -0,0 +1,24 @@
+---
+app-id: org.koromelodev.mindmate
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: MindMate
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --device=dri
+  - --share=network
+modules:
+  - name: MindMate
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

unless it doesn't work.

---

this wasn't adressed.